includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -755,6 +761,36 @@ private function has_used_attributes( $attribute_names ) {
 		return true;
 	}
 
+	/**
+	 * Whether a given class is allowed to be styled in <amp-date-picker>.
+	 *
+	 * That component has child classes that won't be present in the document yet.
+	 * So get whether a class is an allowed child.
+	 *
+	 * @since 1.5.0
+	 * @link https://github.com/airbnb/react-dates/tree/05356/src/components
+	 *
+	 * @param string $class The name of the class to evaluate.
+	 * @return bool Whether the class is allowed as a child of <amp-date-picker>.
+	 */
+	private function is_class_allowed_in_amp_date_picker( $class ) {
+		$class_prefixes = [
```

Comment:
This should be either a const array, or a static variable. Otherwise, you're creating and filling up a fresh new array every single time this method runs.

Reply:
Sure, this is applied in https://github.com/ampproject/amp-wp/pull/4339/commits/01a8e8fc265543052b4ab1858236a12855c4bedd